source/server/session_repository.h

```diff
@@ -1,14 +1,18 @@
 #ifndef NIDEVICE_GRPC_SESSION_REPOSITORY
 #define NIDEVICE_GRPC_SESSION_REPOSITORY
 
-#include <grpcpp/grpcpp.h>
-#include <session.grpc.pb.h>
-
 #include <atomic>
+#include <functional>
+#include <map>
 #include <shared_mutex>
 
 #include "semaphore.h"
 
+namespace grpc {
```

Comment:
I did a little bit of `include` file cleanup and forward-declares etc. Nothing especially made a different, but good policy.